Core/AppRuntime/V8Inspector/Source/V8InspectorSocketServer.cpp

```diff
@@ -351,6 +354,7 @@ namespace Babylon
 
     void InspectorSocketServer::Stop()
     {
+        std::lock_guard<std::mutex> guard{m_mutex};
```

Comment:
What's this for? Is Stop being called from multiple threads?

Reply:
Ah, I missed the PR description. Is this the only function that is called from two threads?

Reply:
It's the only one I've seen so far.